apps/api/src/common/env/env.schema.ts

```diff
@@ -56,7 +56,7 @@ const generalSchema = z.object({
 
   SMTP_HOST: z.string(),
   SMTP_PORT: z.string(),
-  SMTP_EMAIL_ADDRESS: z.string(),
+  SMTP_EMAIL_ADDRESS: z.string().email(),
   SMTP_PASSWORD: z.string(),
   FROM_EMAIL: z.string().regex(/^[a-zA-Z0-9._%+-]+ [<][a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}[>]$/),
```

Comment:
The regex works well for the following formats:
- `name <email@example.com>`
- `your-name <email@example.com>`

However it doesn't recognise `your name <email@example.com>`. Though not mentioned in `.env.example` I am afraid that this is a very common format for sender emails. 

This regex shall take into account for that  as well.

In total the regex shall recognise:

- `name <email@example.com>`
- `your-name <email@example.com>`
- `your name <email@example.com>`

Reply:
Agreed!